packages/@sfx/search-plugin/test/unit/common/search-plugin.test.ts

```diff
@@ -0,0 +1,63 @@
+import { expect, sinon, stub } from '../../utils';
+import SearchPlugin from '../../../src/search-plugin';
+import { SearchPluginOptions, SearchPluginExposedValue } from '../../../src/search-plugin';
+import { BridgeConfig, Query } from 'groupby-api';
+import * as SearchPackage from 'groupby-api';
+
+describe('SearchPlugin', () => {
+  let searchPlugin: any;
```

Comment:
Unless TypeScript complains, we can leave this untyped. It will default to `any`.

```suggestion
  let searchPlugin;
```